### PR TITLE
Added some mentors as a part of GCI

### PR DESCRIPTION
**Pull request by abishekvashok**
Added some mentors as a part of task in GCI for FOSSASIA. Hope you'd accept this PR soon enough. :) Here's the link to my task: https://codein.withgoogle.com/dashboard/task-instances/6273948341764096/?sp-page=1 . Please review it soon  @jig08 @roonyh @sudheesh001 @SaptakS @niccokunzmann @mananwason . Please review and accept(if possible) before any conflicts arrise. The image named anonyms(which is optimized) has been created as I did'nt get the photos of the mentors. Sorry for that :) , but that was the only way to do so.

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://Abhi2424shek.github.io/gci15.fossasia.org

File: _data/mentors.yml [hunk 1]
```diff
@@ -381,9 +381,51 @@
   facebook:
   bio:
   
-- name: Arkodyuti Saha
-  github: arkodyuti
-  image: arkodyuti.jpg
```
Comment (mananwason) [inline _data/mentors.yml:386]: Why have you deleted?

**Comment by mananwason** [comment]
@Abhi2424shek You have just added the names of the Mentor. Please atleast add the github profile.

**Comment by abishekvashok** [comment]
upsss! wait new PR coming right away really sorry for that @mananwason and I will add the github profiles also. Ok @mananwason

**Comment by mananwason** [comment]
and please try to learn how to squash. It's really easy and you should ideally modify code in the same PR. We're are here to help. Thanks!

**Comment by roonyh** [comment]
@Abhi2424shek Agree with @mananwason. You have worked with git and Github enough to learn about  squashing using the git command line tool. Please don't create another PR everytime you need to amend your work.

And I am not sure if the Guy Fawkes mask would be an ideal fit here. Its a well known symbol of an hacktivist group. Lets use something more simple similar to the facebook placeholder profile pic.

**Pull request closed without merging**